rhoc/pkg/cmd/deployments/deployments.go

```diff
@@ -20,7 +22,9 @@ func NewDeploymentsCommand(f *factory.Factory) *cobra.Command {
 		cmd,
 		list.NewListCommand(f),
 		get.NewGetCommand(f),
-		updateChannel.NewUpdateChannelCommand(f))
+		updateChannel.NewUpdateChannelCommand(f),
+		snapshot.NewSnapshotCommand(f),
+		validateSnapshot.NewValidateSnapshotCommand(f))
```

Comment:
no strong opinion but that be split in 2 snapshot subcommands:

  rhoc deployment snapshot create
  rhoc deployment snapshot validate

Reply:
makes sense